content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -23,6 +23,42 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
 Description here.
 
+## ECS Optimizations
+
+<div class="release-feature-authors">authors: @james7132, @JoJoJet</div>
+
+ECS underlies the entire engine, so eliminating overhead in the ECS results in engine-wide speedups. In 0.10, we've found quite a few areas where we were able to massively reduce the overhead and improve CPU utilization for the entire engine.
```

Comment:
```suggestion
ECS underlies the entire engine, so eliminating overhead in the ECS results in engine-wide speedups. In **Bevy 0.10**, we've found quite a few areas where we were able to massively reduce the overhead and improve CPU utilization for the entire engine.
```

The previous blog posts always refer to the new version in boldface, so it's probably best to do it here, too.